src/db/db_build.sql

```diff
@@ -0,0 +1,81 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, posts, messages, chatRooms, diseases, tags, comments;
+
+CREATE TABLE diseases (
+    id SERIAL PRIMARY KEY,
+    diseaseName VARCHAR(100) NOT NULL
+);
+
+INSERT INTO diseases (diseaseName)
+VALUES ('cancer');
+
+CREATE TABLE tags (
+    id SERIAL PRIMARY KEY,
+    tagName VARCHAR(50) NOT NULL,
+    diseaseId INTEGER REFERENCES diseases(id)
+);
+
+INSERT INTO tags (tagName, diseaseId)
+VALUES ('cancer', 1);
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    fullName VARCHAR(40) NOT NULL,
+    email VARCHAR(50) NOT NULL,
+    password VARCHAR NOT NULL,
```

Comment:
you have just `VARCHAR` here without specifying how long it should or how many number of characters should it include

Reply:
because passwords are encrypted how long should we allow them to be?

Reply:
i guess my main question here does it work without specifying the limit?

Reply:
yes then it uses the default limit